### DISCUSSION TITLE
Welcome to Movie_streaming Discussions!

### DISCUSSION THREAD

**Discussion by GoldieBowyien** [GoldieBowyien/Movie_streaming#2] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by NeoDev99** [comment]
Hello guys

**Reply by GoldieBowyien** [reply]
Hello

**Comment by GoldieBowyien** [comment]
Good day guys

**Comment by pheneus** [comment]
lets get it

**Reply by GoldieBowyien** [reply]
okay guys lets get on it.

**Reply by pheneus** [reply]
Thanks!

**Comment by GoldieBowyien** [comment]
We need to work on the read me file

**Reply by pheneus** [reply]
Okay cool

**Comment by pheneus** [comment]
Are we going to create the readme file tomorrow?

**Reply by NeoDev99** [reply]
No, today.
